api/app/signals/apps/signals/models.py

```diff
@@ -607,7 +595,9 @@ class Attachment(CreatedUpdatedModel):
     created_by = models.EmailField(null=True, blank=True)
     _signal = models.ForeignKey(
         "signals.Signal",
-        null=False, on_delete=models.CASCADE
+        null=False,
+        on_delete=models.CASCADE,
+        related_name='attachments',
```

Comment:
This is better than the ModelManager stuff.

Reply:
Yep, it is more in style on how Django gives you functionality out of the box instead of writing your own model managers